src/utils/__tests__/internalLinks-test.js

```diff
@@ -190,11 +200,14 @@ describe('getNarrowFromNarrowLink (part 1)', () => {
     ].forEach(hash => check(hash));
   });
 
-  describe('link containing "is" with valid operand is a special link', () => {
+  describe('"/#narrow/is/<…>" with valid operand is a special link', () => {
     const check = mkCheck(isSpecialNarrow);
-    ['/#narrow/is/private', '/#narrow/is/starred', '/#narrow/is/mentioned'].forEach(hash =>
-      check(hash),
-    );
+    [
+      '/#narrow/is/dm',
+      '/#narrow/is/private',
+      '/#narrow/is/starred',
+      '/#narrow/is/mentioned',
+    ].forEach(hash => check(hash));
```

Comment:
We should also check that each of these give the right specific special narrow. That might best fit among the "part 2" test cases below.